src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -106,16 +147,20 @@ export class CalciteCombobox {
 
   @Event() calciteComboboxChipDismiss: EventEmitter;
 
-  @Listen("calciteComboboxItemChange") calciteComboboxItemChangeHandler(
-    event: CustomEvent<HTMLCalciteComboboxItemElement>
-  ): void {
+  @Listen("click", { target: "document" })
+  documentClickHandler(event: Event): void {
+    const target = event.target as HTMLElement;
+    this.setInactiveIfNotContained(target);
+  }
+
+  @Listen("calciteComboboxItemChange")
+  calciteComboboxItemChangeHandler(event: CustomEvent<HTMLCalciteComboboxItemElement>): void {
```

Comment:
Nitpick: `calciteCombox` is implied so, it could be dropped from the name.

Reply:
I'll leave as is since it wasn't modified.